File: converted/source2/source/zombie/iso/WorldMarkers$Line.java
package zombie.iso;

import zombie.iso.WorldMarkers.Point;

class WorldMarkers$Line {
   Point s;
   Point e;

   WorldMarkers$Line(Point var1, Point var2) {
      this.s = var1;
      this.e = var2;
   }

   @Override
   public String toString() {
      return String.format("{s: %s, e: %s}", this.s.toString(), this.e.toString());
   }
}

File: DoNotUse/include/zombie/iso/WorldMarkers$Line.cpp
#pragma once
#include <string>
#include <vector>
#include <memory>
#include <unordered_map>
#include <unordered_set>
#include <cstdint>
#include "zombie/iso/WorldMarkers/Point.h"

namespace zombie {
namespace iso {


class WorldMarkers {
public:
    Point s;
    Point e;

   WorldMarkers$Line(Point var1, Point var2) {
      this.s = var1;
      this.e = var2;
   }

    std::string toString() {
      return std::string.format("{s: %s, e: %s}", this.s, this.e);
   }
}
} // namespace iso
} // namespace zombie
